src/tests/unit/tests/scanner/custom-rules/css-content-rule.test.ts

```diff
@@ -98,16 +102,21 @@ describe('verify matches', () => {
                 )
                 .verifiable(Times.atLeastOnce());
 
-            testSemantics(divElementFixture, testCaseParameters.testExpectation);
+            withAxeCommonsMocked(
+                'dom',
+                {
+                    isVisible: axeVisibilityMock.object,
+                },
+                () => {
+                    testSemantics(divElementFixture, testCaseParameters.testExpectation);
+                },
+                [getComputedStyleMock],
+            );
         },
     );
 
     function testSemantics(elements: HTMLElement, expectedResult: boolean): void {
-        let result: boolean;
-
-        GlobalScope.using(getComputedStyleMock, axeVisibilityMock).with(() => {
-            result = cssContentConfiguration.rule.matches(elements, null);
-        });
+        const result = cssContentConfiguration.rule.matches(elements, null);
```

Comment:
The helper is only 2 lines long now; I think just repeating it would make the tests more clear about what they're acting/asserting